client/src/main/java/io/pravega/client/tables/KeyValueTableClientConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.tables;
+
+import com.google.common.annotations.Beta;
+import com.google.common.base.Preconditions;
+import java.io.Serializable;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Configuration for the {@link KeyValueTable} client.
+ */
+@Beta
+@Data
+@Builder
+public class KeyValueTableClientConfiguration implements Serializable {
+    private static final long serialVersionUID = 1L;
+    private final int initialBackoffMillis;
+    private final int maxBackoffMillis;
+    private final int retryAttempts;
+    private final int backoffMultiple;
+
+    public static final class KeyValueTableClientConfigurationBuilder {
```

Comment:
The maximum retry wait is 10s (10ms * 2^10). But it will try 10ms, 20ms, 40ms, ...., 1s, 2.5s, 5s, 10s.

That still looks too frequent. I changed the multiplier to 4; that should stagger it a bit.